go/caddy/pfipset/utils.go

```diff
@@ -68,7 +68,7 @@ func updateClusterL2(ctx context.Context, Ip string, Mac string, Network string,
 	logger := log.LoggerWContext(ctx)
 
 	for _, member := range getClusterMembersIps(ctx) {
-		err := post(ctx, "https://"+member.String()+":22223/ipset/mark_layer2/"+Network+"/"+Type+"/"+Catid+"/"+Ip+"/"+Mac+"/1", body)
+		err := post(ctx, "https://"+member.String()+":22223/ipset/mark_layer2/"+Network+"/"+Type+"/"+Catid+"/"+Ip+"/"+Mac+"/?local=1", body)
```

Comment:
This would need to be adjusted for the new request format since it calls the other pfipset daemons

Reply:
And apply the comment above to the other calls in this file